build-tools/packages/build-cli/src/config.ts

```diff
@@ -63,6 +63,23 @@ export interface FlubConfig {
 	 * Configuration for the `generate:releaseNotes` command.
 	 */
 	releaseNotes?: ReleaseNotesConfig;
+
+	/**
+	 * Configuration for `release report` command
+	 */
+	releaseReport?: ReleaseReportConfig;
+}
+
+/**
+ * Configuration for the `release report` command. If this configuration is not present in the config, the
+ * `release report` command will report an error.
```

Comment:
For backcompat, the legacy compat can be set to 0 in the config file and that would fallback to caret. do you think that's okay?

Reply:
I think it'll cause problems if we ever use the command on old branches. Anywhere we want to use it will require a config change. We can merge as is and see if it's a problem in practice.

Reply:
How often do we update the release tools for release branches? Should be easy to update config should we do that.

Reply:
Don't we typically use the in-repo version of build-tools (the one associated with the specific release branch) instead of the latest version? While running a patch release, I encountered an error present in the latest build-tools release, which was resolved by switching to the in-repo version.